app/views/links/edit.html.erb

```diff
@@ -18,7 +18,7 @@
 
 <%= form_for @link, url: local_authority_service_interaction_links_path(@local_authority.slug, @service.slug, @interaction.slug), method: "put" do |f| %>
   <div class="form-group <% if flash[:bad_url] %>has-error<% end %>">
-    <%= f.text_field :url, class: "form-control", value: @link.display_url%>
+    <%= f.text_field :url, class: "form-control", value: @link.url%>
```

Comment:
You probably don't need the `value: @link.url` bit anymore, as it would be the default because `@link` is the object we're using in the form and we're drawing an input field for its `url` attribute.